packages/blade/src/components/Checkbox/__tests__/Checkbox.web.test.tsx

```diff
@@ -3,16 +3,20 @@
 import userEvents from '@testing-library/user-event';
 import React from 'react';
 import { Checkbox } from '../Checkbox';
+import { Link } from '~components/Link';
 import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
 
 describe('<Checkbox />', () => {
   it('should render checkbox with label', () => {
-    const labelText = 'Remember password';
-    const { container, getByText, getByRole } = renderWithTheme(<Checkbox>{labelText}</Checkbox>);
+    const label = (
+      <>
+        I accept <Link>Terms and Conditions</Link>
```

Comment:
Would be better if we add the href in here.

Reply:
done :)